surefire-integration-tests/src/test/java/org/apache/maven/surefire/its/JUnit47ConcurrencyIT.java

```diff
@@ -32,14 +32,14 @@
  *
  * @author Kristian Rosenvold
  */
-public class Junit47concurrencyIT
```

Comment:
Because consistency is a good thing :-) There are some tests named "JunitXXX" and some "JUnitXXX". It's better to have only one naming convention.

Yes I've searched the whole project for occurrences of "concurrentjunit47".